GRR/script_for_delta_part_1.sql

```diff
@@ -1881,19 +1915,31 @@ FROM drug_concept_stage dcs
    AND cc.vocabulary_id LIKE 'RxNorm%'
 WHERE concept_code_1 IS NULL
 AND   dcs.concept_class_id IN ('Ingredient','Brand Name','Dose Form','Supplier')
-AND   cc.invalid_reason IS NULL or cc.invalid_reason = 'U';
+AND   (cc.invalid_reason IS NULL or cc.invalid_reason = 'U');
 
---update relationship_to_concept if targeted concept has 'U'
-UPDATE relationship_to_concept r1
-   SET concept_id_2 = cr.concept_id_2
-FROM relationship_to_concept rtc
+--replace mapping for Updated concepts
+drop table if exists upd_conc;
+create table upd_conc as 
+select distinct concept_code_1,vocabulary_id_1,cr.concept_id_2, 1 
+from relationship_to_concept rtc
```

Comment:
Is there any check whether the target concept is standard ('Concept replaced by' can point to a non-standard one and be followed with another 'Concept replaced by')?